### ISSUE TITLE
Followed the setup instructions, still ':Vader' and :Test' fail

### ISSUE THREAD

**Issue by lestephane** [exercism/vimscript#79]
After i modified .vimrc as shown below i ran `:PlugInstall` without any issue.

```vimrc
" Specify a directory for plugins
" - For Neovim: stdpath('data') . '/plugged'
" - Avoid using standard Vim directory names like 'plugin'
call plug#begin()

Plug 'junegunn/vader.vim'

function! s:exercism_tests()
  if expand('%:e') == 'vim'
    let testfile = printf('%s/%s.vader', expand('%:p:h'),
          \ tr(expand('%:p:h:t'), '-', '_'))
    if !filereadable(testfile)
      echoerr 'File does not exist: '. testfile
      return
    endif
    source %
    execute 'Vader' testfile
  else
    let sourcefile = printf('%s/%s.vim', expand('%:p:h'),
          \ tr(expand('%:p:h:t'), '-', '_'))
    if !filereadable(sourcefile)
      echoerr 'File does not exist: '. sourcefile
      return
    endif
    execute 'source' sourcefile
    Vader
  endif
endfunction

autocmd BufRead *.{vader,vim}
      \ command! -buffer Test call s:exercism_tests()
```

Yet after that when I open vim and issue ':Vader', i get

```text
E492: Not an editor command: Vader
```

Variation of the same for ':Test'

```text
Error detected while processing function <SNR>7_exercism_tests:
line    9:
E492: Not an editor command: Vader <SCRUBBED>/exercism/vimscript/hello-world/hello_world.vader
```

I'm an absolute noob at .vimrc and vim plugin management, what did I miss?

**Comment by mhinz** [comment]
I think you simply miss the `call plug#end()` after your last `Plug` line.

**Comment by mhinz** [comment]
I'm pretty it's the issue, so I'm closing this. :)

So, to make it clear, it should be:

```vim
call plug#begin()
Plug 'junegunn/vader.vim'
call plug#end()

[...]
```

Otherwise, without calling `plug#end()`, the plugins will never get initialised.

**Comment by lestephane** [comment]
Yup, silly me, I stopped reading [the vim-plug install doc](https://github.com/junegunn/vim-plug) after the first `Plug` line, and so I never saw the `call plug#end()` fragment.

I submitted [a PR](https://github.com/exercism/vimscript/pull/80) to simplify the doc (ie saving the user from having to look at other pages to derive what the canonical setup steps are). It's distilled to its essence, I hope.
